flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/legacy/backpressure/BackPressureStatsTrackerImpl.java

```diff
@@ -82,19 +87,21 @@
 	/**
 	 * Creates a back pressure statistics tracker.
 	 *
-	 * @param cleanUpInterval     Clean up interval for completed stats.
-	 * @param backPressureStatsRefreshInterval
+	 * @param coordinator Coordinator for back pressure stats request.
+	 * @param cleanUpInterval Clean up interval for completed stats.
+	 * @param refreshInterval Time interval after which the available back pressure
+	 *                        stats are deprecated and need to be refreshed.
 	 */
 	public BackPressureStatsTrackerImpl(
 			BackPressureRequestCoordinator coordinator,
 			int cleanUpInterval,
-			int backPressureStatsRefreshInterval) {
+			int refreshInterval) {
```

Comment:
Should also check argument for `cleanUpInterval`. Or add `@Nonnegative` for `cleanUpInterval` and `refreshInterval` to replace check?